js/examples/process_file_uri.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Example script to illustrate how to make API calls to the Private AI Docker
+ * container to deidentify text using the enabled classes feature.
+ *
+ * To use this script, please start the Docker container locally, as per the
+ * instructions at https://private-ai.com/docs/installation.
+ *
+ * In order to use the API key issued by Private AI, you can run the script as
+ * `API_KEY=<your key here> node process_file.js` or you can define a `.env`
+ * file which has the line `API_KEY=<your key here>`.
+ */
+const axios = require("axios");
+const dotenv = require("dotenv");
+
+// Use to load the API_KEY for authentication
+dotenv.config();
+
+// Example without async/await
+axios
+  .post(
+    "http://localhost:8080/v3/process/files/uri",
+    {
+      uri: "https://example.com/example-image.jpeg",
+      entity_detection: {
+        accuracy: "high",
+        return_entity: true,
+      },
+      pdf_options: { density: 150 },
+      audio_options: {
+        bleep_start_padding: 0,
+        bleep_end_padding: 0,
+      },
```

Comment:
I know these options will exist by default in the request logic, but it looks kind of weird to explicitly state pdf_options and audio_options for a jpeg on the client's side 🤷